### ISSUE TITLE
1.10.1 breaks dayjs.tz('2021-1-1T00:00:00', 'America/Los_Angeles')

### ISSUE THREAD

**Issue by chengB12** [iamkun/dayjs#1313]
**Describe the bug**
It seems 1.10.1 fixed https://github.com/iamkun/dayjs/issues/1265
which is dayjs.tz('2021-1-1T00:00:00-08:00', 'America/Los_Angeles') issue, however, it introduces new bug

```
import dayjs from 'dayjs'
import utc from 'dayjs/plugin/utc'
import timezone from 'dayjs/plugin/timezone'

dayjs.extend(utc)
dayjs.extend(timezone)

console.log(dayjs.tz('2021-1-1T00:00:00', 'America/Los_Angeles').format())
```
it shows:
2021-01-01T00:00:00Z
**Expected behavior**
We expect it show
2021-01-01T00:00:00-0800
**Information**
 - Day.js Version [e.g. v1.10.1]
 - OS: [e.g. iOS] windows
 - node.js 14.15.0
 - Time zone: [e.g. GMT-08:00 PST (Pacific Daylight Time)]

**Comment by TychoVrahe** [comment]
experiencing the same issue

**Comment by iamkun** [comment]
this has been fixed in #1307
